app/src/main/java/ru/vssemikoz/newsfeed/newsfeed/NewsFeedPresenter.java

```diff
@@ -68,7 +64,7 @@ public void openNewsDetails(int position, Context context) {
     public void changeNewsFavoriteState(int position) {
         NewsItem item = news.get(position);
         item.invertFavoriteState();
-        newsStorage.updateNews(item);
+        newsStorage.updateData(item);
```

Comment:
We shouldn't point to a concrete model in our method name since we already have a context as a class name like `NewsStorage`. So, a method `update` with a `List` parameter will tell us that it will update a list of news.